### PR TITLE
[CHK-1707] - feat(calculate-fees): add payment method description

### PR DESCRIPTION
**Pull request by pietro-tota**
Depends on https://github.com/pagopa/pagopa-infra/pull/1156

#### List of Changes
Add payment method description field into calculate fee response

#### Motivation and Context
Add payment method description field into calculate fee response

#### How Has This Been Tested?
Junit tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Bigfeddy** [comment]
How can we add our bank details

File: src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java [hunk 1]
```diff
@@ -183,19 +183,19 @@ void shouldRetrieveFee() {
         String paymentMethodId = UUID.randomUUID().toString();
         CalculateFeeRequestDto calculateFeeRequestDto = TestUtil.getCalculateFeeRequest();
         BundleOptionDto gecResponse = TestUtil.getBundleOptionDtoClientResponse();
-
+        PaymentMethodDocument paymentMethodDocument = new PaymentMethodDocument(
+                UUID.randomUUID().toString(),
+                "Carte",
+                "Description",
+                PaymentMethodStatusEnum.ENABLED.getCode(),
```
Comment (Bigfeddy) [inline src/test/java/it/pagopa/ecommerce/payment/methods/service/PaymentMethodServiceTests.java:190]: Hey

**Review by Bigfeddy — COMMENTED** [review]
Who's this

**Review by ciuffagianluca — APPROVED** [review]
Well done!

**Review by Fernando-Granato — APPROVED** [review]

**Pull request merged**